src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public abstract class Figure implements Forming {
+    private String name;
+    private String color;
+
+    public Figure(String name, String color) {
+        this.name = name;
+        this.color = color;
+    }
+
+    public abstract double obtainTheArea();

```

Comment:
You should also move it to separate interface interface and then implement it.
https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-abstract-classes-to-set-behaviour-for-classes